README.md

```diff
@@ -150,6 +150,26 @@ The `views` variable should be provided as a list of object with the following k
   }
 }
 ```
+
+### Variable `routines` detailed description
+
+The `routines` variable should be provided as a list of object with the following keys:
+```hcl
+{
+  routine_id = "some_id"                      # The ID of the the routine. The ID must contain only letters, numbers, or underscores. The maximum length is 256 characters.
```

Comment:
Prefixing these values with `return` is probably unnecessary.

Reply:
do you mean variable `return_type`?

Reply:
No I meant that they don't all need to be prefixed with `routine`, sorry for the typo.

Reply:
aah, okay. understood. I'll leave only `routine_id` and `routine_type` as that's how they are named in the actual resource. will remove it from other variables.